Common/Utilities.cs

```diff
@@ -2819,22 +2809,22 @@ public static void UploadFileToWebApp(IPublishingProfile profile, string filePat
                 }))
                 {
                     var fileinfo = new FileInfo(filePath);
-                    ftpClient.LoginAsync().GetAwaiter().GetResult();
+                    ftpClient.LoginAsync().ConfigureAwait(false);

```

Comment:
You need to `await` all these async calls, now. This sample won't work now.

```csharp
await ftpClient.LoginAsync().ConfigureAwait(false);
```

Same for all async calls throughout this PR.

Reply:
Were you reviewing the latest code? i have changed all sync calls with `await`.
This is details in file changed now.
![latest](https://user-images.githubusercontent.com/65536078/90580690-0fb8d100-e1fc-11ea-9552-0a81433823bb.JPG)

Reply:
I was reviewing the latest code at the time I commented, yes. You should still have the `.ConfigureAwait(false)`, though. That's a recommended .NET best practice in *most* cases.

Reply:
Resolved.